percona/percona-everest-cli
# EVEREST-725 dev CI

**DEV pipelines**
---
**Problem:**
EVEREST-107

- check for operator latest module
- check 0.0.0 is used in Makefile and deploy scripts
- update the default codebase state for main branch (version 0.0.0)

more info about the new CI can be found the pipelines doc ticket EVEREST-481 

[EVEREST-107]: https://perconadev.atlassian.net/browse/EVEREST-107?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

Related PRs:
- https://github.com/percona/everest-operator/pull/233
- https://github.com/percona/everest-catalog/pull/2
- https://github.com/percona/percona-everest-backend/pull/374
